### PR TITLE
Refactored Ches to not use class and hopefully work in dev branch

### PR DESCRIPTION
**Pull request by okanji**

### CODE REVIEW

File: ches/emailHandler.ts [hunk 1]
```diff
@@ -33,14 +37,14 @@ export const sendNewRequestEmails = async (formData: PrivateCloudCreateRequestBo
...
-    await chesService.send({
+    await sendEmail({
       body: email,
       // For all project contacts. Sent when the project set deletion request is successfully submitted
       to: [
         request.requestedProject.projectOwner.email,
         request.requestedProject.primaryTechnicalLead.email,
         request.requestedProject.secondaryTechnicalLead?.email,
-      ],
+      ].filter(Boolean),
```
Comment (junminahn) [inline ches/emailHandler.ts:47]: this `filter` is unnecessary in favor of `safeEmails` function inside send function https://github.com/bcgov/platform-services-registry/pull/1306/files#diff-47dbc6dc1d6ea3dc3cd61dd971bc213868f28f32ef8d3618d38b1585ad98981eR90

File: ches/emailHandler.ts [hunk 2]
```diff
@@ -51,14 +55,14 @@ export const sendRequestApprovalEmails = async (request: PrivateCloudRequestWith
...
```
Comment (junminahn) [inline ches/emailHandler.ts:65]: this `filter` is unnecessary in favor of `safeEmails` function inside send function https://github.com/bcgov/platform-services-registry/pull/1306/files#diff-47dbc6dc1d6ea3dc3cd61dd971bc213868f28f32ef8d3618d38b1585ad98981eR90

File: ches/emailHandler.ts [hunk 3]
```diff
@@ -5,19 +5,23 @@ import { NewRequestTemplate } from '@/emails/templates/NewRequestTemplate';
...
-    const send1 = chesService.send({
+    const send1 = await sendEmail({
       body: email,
       // For all project contacts. Sent when the project set deletion request is successfully submitted
-      to: [formData.projectOwner.email, formData.primaryTechnicalLead.email, formData.secondaryTechnicalLead?.email],
+      to: [
+        formData.projectOwner.email,
+        formData.primaryTechnicalLead.email,
+        formData.secondaryTechnicalLead?.email,
+      ].filter(Boolean),
```
Comment (junminahn) [inline ches/emailHandler.ts:20]: this `filter` is unnecessary in favor of `safeEmails` function inside send function https://github.com/bcgov/platform-services-registry/pull/1306/files#diff-47dbc6dc1d6ea3dc3cd61dd971bc213868f28f32ef8d3618d38b1585ad98981eR90

**Review by junminahn — COMMENTED** [review]
Great refactoring 💯

File: ches/index.ts [hunk 4]
```diff
@@ -1,10 +1,113 @@
...
+
+    const data = (await response.json()) as { access_token: string };
+    return data.access_token;
+  } catch (error) {
+    console.error('Exception retrieving token:', error);
+    return null;
+  }
+};
+
+const sendEmail = async (email: Email): Promise<void> => {
```
Comment (junminahn) [inline ches/index.ts:57]: I kind of prefer the `object destruction way of retrieving each property`, but could be a personal choice.
https://github.com/bcgov/platform-services-registry/pull/1306/files#diff-176385a08bc07b6d37488033a729d5ddd96392de93a65659234ef593221c0e10L68

File: ches/index.ts [hunk 5]
```diff
@@ -1,10 +1,113 @@
...
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        bodyType: email.bodyType || 'html',
+        from: email.from || 'Registry <PlatformServicesTeam@gov.bc.ca>',
+        subject: email.subject,
+        body: email.body,
+        to: safeEmails(email.to),
+        // Provide an empty array as a fallback if bcc or cc is undefined
```
Comment (junminahn) [inline ches/index.ts:91]: I believe `safeEmails` handles the empty cases.

**Review by quadruple-j — APPROVED** [review]

**Review by junminahn — APPROVED** [review]

**Pull request merged**